slave.mk

```diff
@@ -638,7 +643,7 @@ ifeq ($(BLDENV),stretch)
 	STRETCH_DOCKER_IMAGES = $(filter $(SONIC_STRETCH_DOCKERS),$(DOCKER_IMAGES_FOR_INSTALLERS) $(EXTRA_DOCKER_TARGETS))
 	STRETCH_DBG_DOCKER_IMAGES = $(filter $(SONIC_STRETCH_DBG_DOCKERS), $(patsubst %.gz,%-$(DBG_IMAGE_MARK).gz, $(STRETCH_DOCKER_IMAGES)))
 else
-	DOCKER_IMAGES := $(SONIC_DOCKER_IMAGES)
+	DOCKER_IMAGES := $(SONIC_BUSTER_DOCKERS)
```

Comment:
keep as it is.

Reply:
With no change to slave.mk and sonic-mgmt-framework setup to be a buster docker, the build fails with the below error.

	make: *** No rule to make target '/redis-tools_5.0.3-3~bpo9+2_amd64.deb', needed by 'target/docker-sonic-mgmt-framework.gz'.  Stop.


After setting up the dependencies so that the correct path is used based on the BLDENV using the below changes,
```
	$(foreach IMAGE,$(DOCKER_IMAGES), $(eval $(IMAGE)_DEBS_PATH := $(DEBS_PATH)))
	$(foreach IMAGE,$(DOCKER_IMAGES), $(eval $(IMAGE)_FILES_PATH := $(FILES_PATH)))
	$(foreach IMAGE,$(DOCKER_DBG_IMAGES), $(eval $(IMAGE)_DEBS_PATH := $(DEBS_PATH)))
	$(foreach IMAGE,$(DOCKER_DBG_IMAGES), $(eval $(IMAGE)_FILES_PATH := $(FILES_PATH)))
```
since DOCKER_IMAGES (SONIC_DOCKER_IMAGES) consists of all the dockers, the dependent debian packages are attempted to be built.

I am seeing a build failure for libsnmp-base_5.7.3+dfsg-1.5_all.deb.

For now, I am filtering out SONIC_STRETCH_DOCKERS from SONIC_DOCKER_IMAGES to build for buster.